Anatomy of a Device.ipynb

```diff
@@ -785,11 +785,10 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
```

Comment:
What motivated this change? I think the original is correct. Ref:

https://raw.githubusercontent.com/spatialaudio/nbsphinx/559fc4e82bc9e2123e546e67b8032643c87cfaf6/doc/allow-errors-per-cell.ipynb

Reply:
please see my comments regarding "tags"